src/Razor/test/Microsoft.AspNetCore.Razor.OmniSharpPlugin.Test/DefaultProjectChangePublisherTest.cs

```diff
@@ -21,12 +22,14 @@ public DefaultProjectChangePublisherTest(ITestOutputHelper testOutput)
     }
 
     [Theory]
-    [InlineData(OmniSharpProjectChangeKind.DocumentAdded)]
-    [InlineData(OmniSharpProjectChangeKind.DocumentRemoved)]
-    [InlineData(OmniSharpProjectChangeKind.ProjectChanged)]
-    public async Task ProjectManager_Changed_EnqueuesPublish(OmniSharpProjectChangeKind changeKind)
+    [InlineData(nameof(OmniSharpProjectChangeKind.DocumentAdded))]
```

Comment:
If I use `OmniSharpProjectChangeKind` directly it wants to make the test internal, which will make it not be found by Xunit.